kuma/spam/akismet.py

```diff
@@ -25,6 +26,13 @@ def __str__(self):
                 (self.status_code, self.debug_help, self.response.text))
 
 
+def akismet_locale(language):
+    """
+    Convert a Django locale to a locale akismet expects. ("en-US" to "en_us")
+    """
+    return translation.to_locale(language).lower()
+
+
 class Akismet(object):
```

Comment:
I think `AkismetFormMixin` might be a good place. The `RevisionForm` is a subclass of that where this is eventually used. Also, this form is where `akismet_parameters` is defined where this is used.